README.md

```diff
@@ -13,27 +13,28 @@ A GitHub App that helps maintain a large number of repositories in a GitHub orga
 `GET /tako/repositories`
 
 ```json
-[
+{
+  "repositories": [
     {
-        "name": "next-foo-bar",
-        "topics": [ "next-app" ]
+      "name": "next-foo-bar"
     },
     {
-        "name": "next-fizz-buzz",
-        "topics": [ "next-serverless" ]
+      "name": "next-fizz-buzz"
     }
-]
+  ]
+}
 ```
 
-`GET /tako/repositories?topic=next-app`
+`GET /tako/repositories/topic/serverless`
```

Comment:
Any reason for this API _not_ to be RESTful?

Reply:
I went with what express supports.

Unfortunately I can't define a param as a separate route, so there'd be one route with two bits of logic.

Wasn't so sure about this one, and happy to change it, but wasn't sure how big the logic for filtering was going to be, so initially wanted to work on the two routes separately.